src/Matrix.php

```diff
@@ -3860,4 +3964,21 @@ public function __toString() : string
     {
         return array_reduce($this->a, [self::class, 'implodeRow'], '') . PHP_EOL;
     }
+
+    /**
+     * Copies elements from a given matrix
+     *
+     * @param \Tensor\Matrix $b
+     * @param int $i
+     * @param int $j
+     * @return void
+     */
+    protected function copyElements(Matrix $b, int $i, int $j): void
+    {
+        for($i1 = 0, $i2 = $i; $i1 < $b->n; ++$i1, ++$i2) {
```

Comment:
Are you able to access the protected property `$b->n` here?

Reply:
yes, PHP allows accessing private/protected properties of the same class inside its methods.

Reply:
Fascinating, I never knew that until now ... does it allow setting the property too or just accessing it?